db/migrate/20170207173837_set_service_display_and_retired_to_false.rb

```diff
@@ -0,0 +1,12 @@
+class SetServiceDisplayAndRetiredToFalse < ActiveRecord::Migration[5.0]
+  Service = Class.new(ActiveRecord::Base)
```

Comment:
if it is not set, then the actual classes (from `app/models`) are loaded in this migration.
Future migrations may freak since the table definition will be cached.

Hmm, it reads like a series of unfortunate events.